.env

```
@@ -0,0 +1 @@
+VUE_APP_DEFAULT_PORT=8001
```

Also, change to the following:

```suggestion
PORT=8001
VUE_APP_DEFAULT_PORT=${PORT}

DEBUG=*
```